plugins/example_plugin/index.js

```diff
@@ -0,0 +1,7 @@
+module.exports = function(app) {
```

Comment:
@starsinmypockets I would call the plugin just `example` since it is inside the `plugins` directory :smile: